docs/18_github_actions_ci_integration.md

```diff
@@ -0,0 +1,229 @@
+# CI Integrations GitHub Actions integration.
+
+> Summary of the proposed change
+
+Describe the integration mechanism of the CI Integrations component to the GitHub Actions to automatically push the build data.
+
+# References
+
+> Link to supporting documentation, GitHub tickets, etc.
+
+- [CI integrations](metrics/ci_integrations/docs/01_ci_integration_module_architecture.md)
+
+# Motivation
+
+> What problem is this project solving?
+
+This document describes the configuration process of the CI Integrations to automatically synchronize the GitHub Actions build data to the Metrics Web application.
+
+# Goals
+
+> Identify success metrics and measurable goals.
+
+This document aims the following goals: 
+
+- Explain the CI Integrations configuration process.
+- Explain the mechanism of automation build imports using the GitHub Actions and CI Integrations tool.
+
+# Non-Goals
+
+> Identify what's not in scope.
+
+This document does not describe the configuration of building or publishing jobs.
+
+# Design
```

Comment:
```suggestion
# CI Integration tool configuration
```
I think we are free to adjust template sections as we see fit (it's more descriptive)